<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="top_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="pharmacy_name">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>20</pointsize>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Pharmacy Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pharmacy_address">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Pharmacy Address, City Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="pan_and_dda">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="pan_no">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PAN No.: &lt;PAN No. Here&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dda_no">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>DDA No.: &lt;DDA No. Here&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stacked_widget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page_add_item"/>
      <widget class="QWidget" name="page_add_batch"/>
      <widget class="QWidget" name="page_get_all_items"/>
      <widget class="QWidget" name="page_add_bill"/>
      <widget class="QWidget" name="page_get_batches"/>
      <widget class="QWidget" name="page_get_all_bills"/>
      <widget class="QWidget" name="page_edit_item"/>
      <widget class="QWidget" name="page_edit_batch"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_item">
    <property name="title">
     <string>Items</string>
    </property>
    <addaction name="action_get_all_items"/>
    <addaction name="action_add_item"/>
    <addaction name="action_edit_item"/>
   </widget>
   <widget class="QMenu" name="menu_batch">
    <property name="title">
     <string>Batches</string>
    </property>
    <addaction name="action_get_batches"/>
    <addaction name="action_add_batch"/>
    <addaction name="action_edit_batch"/>
   </widget>
   <widget class="QMenu" name="menu_bill">
    <property name="title">
     <string>Bills</string>
    </property>
    <addaction name="action_get_all_bills"/>
    <addaction name="action_add_bill"/>
   </widget>
   <addaction name="menu_item"/>
   <addaction name="menu_batch"/>
   <addaction name="menu_bill"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_get_all_items">
   <property name="text">
    <string>List All Items</string>
   </property>
  </action>
  <action name="action_add_item">
   <property name="text">
    <string>Add Item</string>
   </property>
  </action>
  <action name="action_edit_item">
   <property name="text">
    <string>Edit Item</string>
   </property>
  </action>
  <action name="action_remove_item">
   <property name="text">
    <string>Remove Item</string>
   </property>
  </action>
  <action name="action_get_batches">
   <property name="text">
    <string>List Batches</string>
   </property>
  </action>
  <action name="action_add_batch">
   <property name="text">
    <string>Add a Batch</string>
   </property>
  </action>
  <action name="action_edit_batch">
   <property name="text">
    <string>Edit a Batch</string>
   </property>
  </action>
  <action name="action_remove_batch">
   <property name="text">
    <string>Remove a Batch</string>
   </property>
  </action>
  <action name="action_get_all_bills">
   <property name="text">
    <string>List All Bills</string>
   </property>
  </action>
  <action name="action_add_bill">
   <property name="text">
    <string>Add Bill</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
